src/spyglass/utils/dj_helper_fn.py

```diff
@@ -35,16 +35,35 @@
 
 
 def ensure_names(
-    table: Union[str, Table, Iterable] = None
+    table: Union[str, Table, Iterable] = None, force_list: bool = False
 ) -> Union[str, List[str], None]:
-    """Ensure table is a string."""
+    """Ensure table is a string.
+
+    Parameters
+    ----------
+    table : Union[str, Table, Iterable], optional
+        Table to ensure is a string, by default None. If passed as iterable,
+        will ensure all elements are strings.
+    force_list : bool, optional
+        Force the return to be a list, by default False, only used if input is
+        iterable.
+
+    Returns
+    -------
+    Union[str, List[str], None]
+        Table as a string or list of strings.
+    """
+    # is iterable (list, set, set) but not a table/string
+    is_collection = isinstance(table, Iterable) and not isinstance(
+        table, (Table, TableMeta, str)
+    )
+    if force_list and not is_collection:
```

Comment:
Just a check, in the case of `force_list=True` and `table is None` the return will be `[None,]` rather than `[ ]`. Is that an issue to place `None` in the `banned_search_tables` set?

Reply:
In distance restrictions, the next-up table is checked against the list and skipped if in that list. If the queuing func returns 'None', I would want to skip that too, but I don't think it would

[Check](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/utils/dj_graph.py#L1042)

[Queuing func](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/utils/dj_graph.py#L346)